File: issuetracker/src/main/java/hu/elte/issuetracker/model/Label.java
package hu.elte.issuetracker.model;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.sun.istack.NotNull;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.List;

@Entity
@Data
@EqualsAndHashCode
@AllArgsConstructor
@NoArgsConstructor
public class Label {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Column
    @NotNull
    private String text;

    @JsonIgnore
    @ManyToMany(mappedBy = "labels")
    private List<Issue> issues;
}

File: issuetracker-ts/src/entities/label.ts
import {
  Collection,
  Entity,
  ManyToMany,
  PrimaryKey,
  Property,
} from '@mikro-orm/core';
import { Issue } from './issue';

@Entity()
export class Label {
  @PrimaryKey()
  id!: number;

  @Property()
  text!: string;

  @ManyToMany(() => Issue, 'labels')
  issues = new Collection<Issue>(this);
}
